SocketIoClient.cpp

```diff
@@ -92,8 +92,23 @@ void SocketIoClient::emit(const char* event, const char * payload) {
 	_packets.push_back(msg);
 }
 
+void SocketIoClient::remove(const char* event) {
+	auto e = _events.find(event);
+
+	if(e != _events.end()) {
+		_events.erase(_events.find(event));
```

Comment:
could this be? ```_events.erase(e)```?

Reply:
Yes of course, I saw that today revising my code. Thanks